modules/grafana/files/dashboards_aws/aws-amazonmq.json

```diff
@@ -0,0 +1,382 @@
+{
+  "__requires": [
+    {
+      "type": "grafana",
+      "id": "grafana",
+      "name": "Grafana",
+      "version": "4.5.2"
+    },
+    {
+      "type": "panel",
+      "id": "graph",
+      "name": "Graph",
+      "version": ""
+    },
+    {
+      "type": "datasource",
+      "id": "cloudwatch",
+      "name": "CloudWatch",
+      "version": "1.0.0"
+    }
+  ],
+  "annotations": {
+    "list": []
+  },
+  "editable": true,
+  "gnetId": null,
+  "graphTooltip": 0,
+  "hideControls": false,
+  "id": null,
+  "links": [],
+  "refresh": "10s",
+  "rows": [
+    {
+      "editable": true,
```

Comment:
Was this line required, or was this the one I added in when I forgot you had to be signed in to edit 🙃 ?

Reply:
This line has always been there, AFAICT - all the other dashboards have `"editable": true` lines, so I suggest we stick with the convention